src/app/core/learning-object-module/search/search.service.ts

```diff
@@ -75,4 +78,8 @@ export class SearchService {
       return throwError(error);
     }
   }
+
+  registerChange() {
+    this.needsChange$.next();
+  }
```

Comment:
Where is this called in the service?

Reply:
Only reference is in the add-evaluator.component.ts file. Should I move this to that file since its not used anywhere else?
<img width="389" alt="Screenshot 2024-10-10 at 2 16 51 PM" src="https://github.com/user-attachments/assets/373a7f01-5652-41c2-9bb3-29e45c9d3fdf">

Reply:
I would see who is subscribed to this and then make the call.